casbin/CMakeLists.txt

```diff
@@ -74,7 +74,8 @@ set(CMAKE_CXX_STANDARD 17)
 
 add_library(casbin STATIC ${CASBIN_SOURCE_FILES})
 
-target_precompile_headers(casbin PRIVATE pch.h)
+# target_precompile_headers(casbin PRIVATE pch.h)
```

Comment:
remove completely, if we need anything we have version control ;)

Reply:
this change solves https://packboard.atlassian.net/browse/FIR-12533